java/integTest/src/test/java/glide/SharedCommandTests.java

```diff
@@ -1011,4 +1011,23 @@ public void zcard(BaseClient client) {
                 assertThrows(ExecutionException.class, () -> client.zcard("foo").get());
         assertTrue(executionException.getCause() instanceof RequestException);
     }
+
+    @SneakyThrows
+    @ParameterizedTest
+    @MethodSource("getClients")
+    public void zpopmax(BaseClient client) {
+        String key = UUID.randomUUID().toString();
+        Map<String, Double> membersScores = Map.of("a", 1.0, "b", 2.0, "c", 3.0);
+        assertEquals(3, client.zadd(key, membersScores).get());
+        assertEquals(Map.of("a", 1.0), client.zpopmin(key).get());
+        assertEquals(Map.of("b", 2.0, "c", 3.0), client.zpopmin(key, 3).get());
```

Comment:
```suggestion
        assertEquals(Map.of("b", 2.0, "c", 3.0), client.zpopmin(key, 10).get());
```

Reply:
I don't get what you are trying to accomplish via 10?

Reply:
Ignore me... 3 is good enough.